qaframework-bdd-tests/src/test/resources/features/refapp-2.x/09-form/forms.feature

```diff
@@ -1 +1,12 @@
-Feature: Forms Management
\ No newline at end of file
+Feature: Form Management
+
+  Background:
+  Given  a user click on manage html forms
```

Comment:
what happens when the user click on manage html forms?

Reply:
> what happens when the user click on manage html forms?

Its the initial point that tracks the page functionality. Let be little detailed.Issue to do with forms. Each single run must first pass through manage html forms to do two things one. First tracks the  avaiability of forms. if the forms is available. then it simply go to home page.

Reply:
I expect when a user click on manage htlm form link/app then a certain page is loaded, is this the ideal ?

Reply:
Yeah that's the deal still and i it does the same.

Reply:
don't you think we need a gherkin step that affirms the page is loaded successfully?

Reply:
Added that scenario. at first i wasnt seeing it as a neccessary

Reply:
Resolved thanks.